.github/workflows/testing.yml

```diff
@@ -444,12 +444,31 @@ jobs:
           git clone -b v3.4.16 https://github.com/etcd-io/etcd.git
           cd etcd
           ./build
-          export PATH="$PATH:`pwd`/bin"
+          echo "`pwd`/bin"
+          echo "`pwd`/bin" >> $GITHUB_PATH
```

Comment:
?

Reply:
This is how you add things to PATH in github actions. Its just making the etcd and etcdctl executables available in other steps

Reply:
Maybe add an echo above these two that says
echo "Adding to Github Actions Path:"
echo "`pwd`/bin"

Reply:
Done